src/features/movies/MoviePage/MovieDetails/index.js

```diff
@@ -27,9 +25,7 @@ export const MovieDetails = (props) => {
         <Wrapper>
             <Title>{props.title}</Title>
             <Year dateTime={props.date}>{useFormatDate(props.date, "year")}</Year>
-            <Poster>
-                {(props.image && <Image src={props.image} />) || <CameraIcon />}
-            </Poster>
+            <Image src={props.image} alt={props.title} />
```

Comment:
This could still give error 404 on console when poster_path from API is null.

Reply:
i fix that just moment before